packages/react-query/src/useQuery.ts

```diff
@@ -81,6 +106,20 @@ export function useQuery<
   >,
 ): UseQueryResult<TData, TError>
 
+export function useQuery<
+  TQueryFnData = unknown,
+  TError = unknown,
+  TData = TQueryFnData,
+  TQueryKey extends QueryKey = QueryKey,
+>(
+  queryKey: TQueryKey,
+  queryFn: QueryFunction<TQueryFnData, TQueryKey>,
+  options: Omit<
```

Comment:
`options` needs to be defined for this overload, otherwise, typescript will require `options` to be at least an empty object to fallback to the following overloads